.github/workflows/validate.yaml

```diff
@@ -0,0 +1,42 @@
+name: JSON Schema Validation
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  validate:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # For git diff
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 'stable'
+
+      - name: Install osv-linter
+        run: |
+          git clone https://github.com/ossf/osv-schema.git -b v1.6.7
+          cd ./osv-schema/tools/osv-linter 
+          go build -o ../../../osv-linter ./cmd/osv
+          cd ../../..
+          rm -rf osv-schema
+
+      - name: Validate JSON
+        run: |
+          CHANGED_FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} | grep .json) && : // Ignore errors
+          if [ -n "$CHANGED_FILES" ]; then
+            echo "Validating JSON files: $CHANGED_FILES"
+            ## osv-linter doesn't return error code for invalid JSON. So we should handle this case.
+            output=$(./osv-linter record lint vulns/AQUA-2024-0001.json 2>&1 || true)
```

Comment:
Need to use `$CHANGD_FILES` instead of AQUA-2024-0001.json.

Reply:
Thanks, I forgot to change that after testing on my local PC.

Fixed in https://github.com/aquasecurity/vuln-list-aqua/pull/2/commits/81d7fddebfb071bcff03cd8b81d914ebac02762d

Reply:
Can `osv-linter` take several files as its argument?

Reply:
`osv-linter` works correctly with multiple files - https://github.com/DmitriyLewen/vuln-list-aqua/actions/runs/12426310367/job/34694377977